### PR TITLE
Add database to keycloak-deployment

### LINKED ISSUE #46: keycloak-deployment is missing a database
The manifests in `base/keycloak-deployment` are missing a database causing Keycloak to loose data on pod restart which causes subsequent runs of Kuadant testsuite to fail.

I suggest to add database for Keycloak. The database should:
- Be deployed alongside the Keycloak
- Work on Kind as well as on Openshift
- Use PersisentVolume

### PR DESCRIPTION
**Pull request by azgabur**
Closes #46

### CODE REVIEW

File: base/keycloak-deployment/deployment.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ spec:
       labels:
         app: keycloak
     spec:
+      imagePullSecrets: 
+        - name: pull-
```
Comment (pehala) [inline base/keycloak-deployment/deployment.yaml:17]: is that really needed?

File: base/keycloak-deployment/db.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      app: keycloak-db
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: keycloak-db
+    spec:
+      containers:
+        - name: keycloak-db
+          image: quay.io/centos7/postgresql-10-centos7
```
Comment (pehala) [inline base/keycloak-deployment/db.yaml:19]: Why are we using postgres 10, cannot we use newer version?

File: base/keycloak-deployment/deployment.yaml (hunk 1 above)
Comment (azgabur) [reply base/keycloak-deployment/deployment.yaml:17]: Hoops, leftover from previous iteration. Will remove it

File: base/keycloak-deployment/db.yaml (hunk 2 above)
Comment (azgabur) [reply base/keycloak-deployment/db.yaml:19]: I used Postgress 10 as seen in `base/rhbk/rhbk-db.yaml`.
I changed the image for newer.

**Review by ficap — APPROVED** [review]

**Pull request merged**